app/models/image.rb

```diff
@@ -0,0 +1,6 @@
+class Image < ApplicationRecord
+  validates :url, presence: true
+  validates :source, presence: true
+
+  has_one_attached :original
```

Comment:
We talked about this yesterday, but I'm just leaving us this note to remember to add an attached glitched image too. :)